source/blog/2018-06-30-organizing-our-contributors.md

```diff
@@ -0,0 +1,48 @@
+
+All that as it is, we also can’t lose our central coordination point, so along with these reorganized teams, we’re establishing the Ember Steering Committee. The Steering Committee will take the lead on community support responsibilities that we’re previously under the Core Team’s purview, while the newly-refined Ember.js Core sticks with the technical leadership responsibilities. 
+
+The Steering Committee will be responsible for things including, but not limited to:
+
+- Owning the [Community Guidelines](https://emberjs.com/guidelines/) and helping people stick to them
+- Serving as the public-facing ombudsman
+- Organizing events and conferences
+- Supporting local meetup organizers
+- Managing Ember's brand and visual identity
+- Facilitating cross-team coordination
+- Owning meta-level work, like making improvements to the RFC process itself
+- Establishing a framework and processes for each team to thoughtfully manage its own membership
+- Dealing with policy/legal questions
+
+The Ember.js Core Team will retain its traditional technical leadership responsibilities around the Core Ember.js and Glimmer projects, including but not limited to:
+
+- Sorting out which RFCs have enough consensus to move forward
+- Merging PRs
+- Managing the release process
+- Promoting overall ecosystem compatibility and upgradeability
+- Handling security issues
+
+ The first concrete example of how the Steering Committee can serve as facilitators for community-wide projects is the EmberJS2018 road-mapping process. Everyone on Core is committed to making sure all that great feedback from the community doesn't get dropped or forgotten, and the Steering Committee is taking responsibility for making sure a clear community roadmap is synthesized. It's not the Steering Committee's job to set the items on that roadmap—that's going to emerge from work by all of the Core teams—but it's the Steering Commitee's job to make sure the work happens, by establishing helpful processes and leading the charge.
+
+The initial membership of the Steering Committee comes from people on Ember.js Core who were already working on community support tasks: Leah Silber, Melanie Sumner, Yehuda Katz, Tom Dale, Edward Faulkner and Katie Gengler.
+
+How will this change impact the wider community? Hopefully you’ll notice both jobs being done more effectively than before. Look for more great things from coming soon, including a ton of framework features landing in the 3.x series, and a refresh of the public website and messaging.
+
+Lastly, Ember.js would not be what it is without a huge community of leaders: addon maintainers, strike team members, RFC authors, meetup organizers, and more. And with respect to the duties now reorganized under the steering committee, specifically to Leah Silber, who has been managing most of these responsibilities on her own as part of her Ember Core Team responsibilities since first organizing the very first Ember logo and website way back in 2011. 
```

Comment:
And *especially* with respect to the duties ?

Not sure what we're after here, but it's a bit bumpy at the moment ...